.github/workflows/unix_linux.yml

```diff
@@ -57,26 +83,16 @@ jobs:
             HDF5_PATH=/usr/local/HDF_Group/HDF5/1.12.0
           fi
 
-          if [[ "${{ matrix.os }}" == "ubuntu"*  ]]; then
+          if [ "${{ matrix.os }}" == "debian" ] || [ "${{ matrix.os }}" == "ubuntu" ]; then
```

Comment:
Do we need a conditional still if we remove Mac? Or we add some failure path if we don't match this condition

Reply:
now removed this condition. Perhaps we will bring this back one day it more OS get added